app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
+
+  def most_recent_posts
+    posts[0, 3]
+  end

```

Comment:
- [ ] to be sure you're getting the correct posts you should order them first, which will look something like the below.
```
posts.order('created_at Desc').limit(3)
```